File: crates/codegraph-bench/fixtures/mini/tools/greeter.py
class Greeter:
    def __init__(self, prefix: str) -> None:
        self.prefix = prefix

    def greet(self, name: str) -> str:
        return f"{self.prefix}, {name}"


def make_greeting(name: str) -> str:
    greeter = Greeter("hello")
    return greeter.greet(name)

File: crates/codegraph-extract/tests/fixtures/markup_risk/Greeter.swift
import Foundation

public class Greeter: Base, Drawable {
    @Published var name: String = "hi"

    public static func make() -> Greeter? { return Greeter() }

    func greet(name: String) -> String {
        return helper(name)
    }
}

struct Point { var x: Int }

enum HTTPMethod { case put, delete }

protocol Drawable { func draw() }

typealias Alias = Greeter

extension KF.Builder {
    func build() {}
}

func topLevel() async {
    Greeter.make().greet(name: "x")
    session.request(url)
}

let answer = 42
